common/uri/src/main/java/io/helidon/common/uri/UriQueryWriteable.java

```diff
@@ -42,13 +42,13 @@ static UriQueryWriteable create() {
     UriQueryWriteable set(String name, String... value);
 
     /**
-     * Add a new query parameter or add a value to existing.
+     * Add a new query parameter or add values to existing.
      *
      * @param name  name of the parameter
-     * @param value additional value of the parameter
+     * @param value additional value(s) of the parameter
      * @return this instance
      */
-    UriQueryWriteable add(String name, String value);
+    UriQueryWriteable add(String name, String... value);
```

Comment:
add is intentionally using a single parameter, as it adds to existing values. If you want to create a multi valued query parameter, use #set(String...).
Please revert back to original

Reply:
reverted